ng-dev/pr/merge/defaults/integration.spec.ts

```diff
@@ -104,15 +106,17 @@ describe('default target labels', () => {
   async function getBranchesForLabel(
     name: string,
     githubTargetBranch = 'master',
-    labels?: TargetLabel[],
   ): Promise<string[] | null> {
-    if (labels === undefined) {
-      labels = await computeTargetLabels();
-    }
+    const labels = await computeTargetLabels();
+    spyOn(labelDefaults, 'getTargetLabels').and.resolveTo(labels);
     let label: TargetLabel;
     try {
-      label = getTargetLabelFromPullRequest({labels}, [name]);
+      label = await getTargetLabelFromPullRequest({}, [name]);
     } catch (error) {
+      console.log(error);
+      if (error.message === 'This repository does not use target labels') {
+        return [githubTargetBranch];
```

Comment:
This should be removed now. I think it's a dead path. We should fully rely on the actual logic from the merge script.